src/components/calcite-notice/calcite-notice.tsx

```diff
@@ -26,7 +26,7 @@ import { CSS_UTILITY } from "../../utils/resources";
  * @slot title - Title of the notice (optional)
  * @slot message - Main text of the notice
  * @slot link - Optional action to take from the notice (undo, try again, link to page, etc.)
- * @slot action-end - Allows adding a `calcite-action` at the end of the notice
+ * @slot action-end - Allows adding a `calcite-action` at the end of the notice. It is recommended to use 2 or less actions.
```

Comment:
Doesn't this need to change to `actions-end`?

Reply:
Maybe we need to think about checking generated doc in tests? :)

Reply:
Thanks for catching that. 😅 

I'm treating this as a new feature since this change was omitted from changelog and the slot's name wasn't matching either. 

> Maybe we need to think about checking generated doc in tests? :)

That'd be tricky, but definitely a good idea. I'll create a research issue for this.

Reply:
If we build stories from the json-doc then this would be caught there most likely.